src/core_codemods/remove_module_global.py

```diff
@@ -0,0 +1,22 @@
+import libcst as cst
+from libcst.metadata import GlobalScope, ScopeProvider
+from typing import Union
+from codemodder.codemods.api import BaseCodemod, ReviewGuidance
+from codemodder.codemods.utils_mixin import NameResolutionMixin
+
+
+class RemoveModuleGlobal(BaseCodemod, NameResolutionMixin):
+    NAME = "remove-module-global"
+    SUMMARY = "Remove Module-level Global Call"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
+    DESCRIPTION = "Remove Lines with `global` keyword at Module Level"
```

Comment:
```suggestion
    DESCRIPTION = "Remove `global` keyword at module level"
```

Reply:
The previous comment fixed the reference to "call" which isn't correct because it's not a call. The description in both places could be "Remove `global` usage at module level".